corehq/apps/es/migration_operations.py

```diff
@@ -23,6 +23,19 @@ def run(self, *args, **kw):
     def reverse_run(self, *args, **kw):
         raise NotImplementedError(type(self).__name__)
 
+    def _is_mapping_incompatibe(self, mapping_es_major_versions):
```

Comment:
I first read this as _is mapping compatible_, which is the opposite of what it really says. Also, the negative "incompatible" makes it harder to understand the return value:
- True: not compatible (positve -> negative)
- False: compatible (negative -> positive)

What do you think of changing this name to `_is_mapping_compatible` and reversing the return value? 

Alternately, maybe the name could be changed to something like `_should_skip_operation`, which would not require changing the boolean logic and is positive -> positive / negative -> negative?
- True: skip
- False: do not skip

Reply:
Yeah, it can get confusing. I like `_should_skip_operation` name, would go with it.

Reply:
https://github.com/dimagi/commcare-hq/pull/33468/commits/2dbfab9c08390dcd9fcc96ff4c41d79d2b08c8b3